src/Pages/CourseDetail.elm

```diff
@@ -1477,6 +1477,13 @@ filterExam exams enrollments =
     List.map (\( use, id, exam ) -> ( id, exam ))
         (List.filter (\( use, id, exam ) -> use) examSuccess)
 
+type alias MaybePoints =
```

Comment:
Why is it called `MaybePoints`? As Elm uses `Maybe` for optionals I would've expected a second `Nothing` variant. Also we have the same definition in `src/Pages/SheetDetail.elm`. Can we maybe add it to `src/Types.elm` or some other place where datatype are collected?

Reply:
Sounds good to me. I renamed it to `SheetPointsAchievable` and moved it to `src/Types.elm`.